### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by DACE-DL**
Thank you in advance

### CODE REVIEW

File: LPT/.htaccess [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+
+RewriteRule ^$ https://dace-dl.github.io/ [R=302,L]
+
+RewriteRule ^Ontology$ https://dace-dl.github.io/download/ [R=302,L]
+RewriteRule ^Ontology/$ https://dace-dl.github.io/download/ [R=302,L]
+RewriteRule ^Ontology/(.*)$ https://dace-dl.github.io/download/#$1 [NE,R=302,L]
```
Comment (dgarijo) [inline LPT/.htaccess:10]: This rule generalizes the previous one. It's not necessary

**Comment by dgarijo** [comment]
Please add yourself as part of the maintainers in the readme

File: LPT/readme.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ LPT Classification
 * Ontology <https://w3id.org/LPT/Ontology>
 * Thesaurus <https://w3id.org/LPT/Thesaurus>
 Contacts:
```
Comment (TallTed) [inline LPT/readme.md:10]:
```suggestion

Contacts: 
```

**Comment by dgarijo** [comment]
@DACE-DL I still you have not added yourself as part of the maintainer list. Please add your github  handle in the readme file

**Pull request merged**